src/PluginClient.php

```diff
@@ -79,7 +78,7 @@ public function sendRequest(RequestInterface $request)
         $pluginChain = $this->createPluginChain($this->plugins, function (RequestInterface $request) {
             try {
                 return new FulfilledPromise($this->client->sendRequest($request));
-            } catch (HttplugException $exception) {
+            } catch (\Exception $exception) {
```

Comment:
How do we move forward? How do we avoid issues like this https://github.com/php-http/client-common/pull/33 to occur?

We have to make sure that no plugin ever receives anything else than a HTTPlug exception.